static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx

```diff
@@ -429,9 +433,24 @@ function getSpanEvidenceValue(span: Span | null): string {
     return span.op;
   }
 
+  if (span.op === 'db' && span.description) {
+    return (
+      <StyledCodeSnippet language="sql">
+        {`${span.op} - ${formatter.toString(span.description)}`}
+      </StyledCodeSnippet>
+    );
+  }
+
   return `${span.op} - ${span.description}`;
 }
 
+const StyledCodeSnippet = styled(CodeSnippet)`
+  pre {
+    /* overflow is set to visible in global styles so need to enforce auto here */
+    overflow: auto !important;
```

Comment:
Makes sense, but watch out because I've gotten guidance that wrapping code in the UI might be misleading. It'll make it seems that the code that newlines that weren't there. That said, this is _formatted_ code, so why not? TBH I should finish my formatter and have it split up long lines 🤔

Reply:
oh it won't wrap, it'll scroll

Reply:
Oh I misread, my mistake!